catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/image/ImageInputTransformerTest.java

```diff
@@ -686,7 +668,9 @@ private InputStream getInputStream(String filename) {
     map.put(ImageAttribute.FILE_PART_TYPE_ATTRIBUTE, "IM");
     map.put(ImageAttribute.IMAGE_IDENTIFIER_1_ATTRIBUTE, "Missing ID");
     map.put(ImageAttribute.TARGET_IDENTIFIER_ATTRIBUTE, null);
-    map.put(ImageAttribute.IMAGE_IDENTIFIER_2_ATTRIBUTE, "- BASE IMAGE -");
+    final String imageIdentifier2 = "- BASE IMAGE -";
+    map.put(ImageAttribute.MISSION_ID_ATTRIBUTE, imageIdentifier2);
+    map.put(ImageAttribute.IMAGE_IDENTIFIER_2_ATTRIBUTE, imageIdentifier2);
```

Comment:
@peterhuffer, following the convention for the other attributes in the `ImageAttribute` class, I added this line to make sure that the `isr.mission-id` attribute is still unit tested even though it was moved.